angular/core/models/user_model.coffee

```diff
@@ -28,6 +28,19 @@ angular.module('loomioApp').factory 'UserModel', (BaseModel, AppConfig) ->
     parentGroups: ->
       _.filter @groups(), (group) -> group.isParent()
 
+    topLevelGroups: ->
```

Comment:
Do we use `parentGroups` anymore, or is this replacing it? I'd much prefer this logic was called `parentGroups`, and that we didn't introduce a new concept here.